src/rgw/rgw_op.cc

```diff
@@ -5276,6 +5276,30 @@ void RGWCompleteMultipart::execute()
   meta_obj.set_in_extra_data(true);
   meta_obj.index_hash_source = s->object.name;
 
+  rgw_pool meta_pool;
+  rgw_raw_obj raw_obj;
+  librados::ObjectWriteOperation op;
+  librados::IoCtx ioctx;
+  rados::cls::lock::Lock l("RGWCompleteMultipart");
+  int max_lock_secs_mp = s->cct->_conf->rgw_mp_lock_max_time;
```

Comment:
we shouldn't be adding new config variables to `legacy_config_opts.h`. instead, we should rely on the version in `options.cc` and access it by name with `s->cct->_conf->get_val<int64_t>("rgw_mp_lock_max_time")`